docker.stats() does not work with docker 1.13.1 and centos7

## Description

dockerClient.stats() results in exception

*Note: Works fine in ubuntu-based distros. Does not in CentOS7.
*Note: 7.0.2 did not work for SOME ubuntu-based distros and 8.0.0 fixed that, but it still does not with Centos.

## How to reproduce

DockerClient docker = DefaultDockerClient.fromEnv().build();
docker.stats("containerId");

## What do you expect

to get a ContainerStats object

## What happened instead

Got an DockerRequestException

## Software:

- `docker version`: 
Client:
 Version:      1.13.1
 API version:  1.26
 Go version:   go1.7.5
 Git commit:   092cba3
 Built:        Wed Feb  8 06:38:28 2017
 OS/Arch:      linux/amd64

Server:
 Version:      1.13.1
 API version:  1.26 (minimum version 1.12)
 Go version:   go1.7.5
 Git commit:   092cba3
 Built:        Wed Feb  8 06:38:28 2017
 OS/Arch:      linux/amd64
 Experimental: false

- Spotify's docker-client version: 8.0.0

## Full backtrace

```text
Feb 23 03:59:36 slave1.lc java[2060]: com.spotify.docker.client.exceptions.DockerRequestException: Request error: GET unix://localhost:80/containers/23df6faa3bbecd531c2e28307eb4f5719d116aafb05321462fdb583169c49012/stats?stream=0: 200
Feb 23 03:59:36 slave1.lc java[2060]: at com.spotify.docker.client.DefaultDockerClient.propagate(DefaultDockerClient.java:2163) ~[metrics.jar:DEV]
Feb 23 03:59:36 slave1.lc java[2060]: at com.spotify.docker.client.DefaultDockerClient.request(DefaultDockerClient.java:2102) ~[metrics.jar:DEV]
Feb 23 03:59:36 slave1.lc java[2060]: at com.spotify.docker.client.DefaultDockerClient.stats(DefaultDockerClient.java:1831) ~[metrics.jar:DEV]
Feb 23 03:59:36 slave1.lc java[2060]: at com.infra.docker.StatsCollectorScheduler.exec(StatsCollectorScheduler.java:56) ~[metrics.jar:DEV]
Feb 23 03:59:36 slave1.lc java[2060]: at com.infra.docker.ScheduledService$1.run(ScheduledService.java:31) [metrics.jar:DEV]
Feb 23 03:59:36 slave1.lc java[2060]: at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) [na:1.8.0_77]
Feb 23 03:59:36 slave1.lc java[2060]: at java.util.concurrent.FutureTask.runAndReset(FutureTask.java:308) [na:1.8.0_77]
Feb 23 03:59:36 slave1.lc java[2060]: at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(ScheduledThreadPoolExecutor.java:180) [na:1.8.0_77]
Feb 23 03:59:36 slave1.lc java[2060]: at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:294) [na:1.8.0_77]
Feb 23 03:59:36 slave1.lc java[2060]: at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142) [na:1.8.0_77]
Feb 23 03:59:36 slave1.lc java[2060]: at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617) [na:1.8.0_77]
Feb 23 03:59:36 slave1.lc java[2060]: at java.lang.Thread.run(Thread.java:745) [na:1.8.0_77]
Feb 23 03:59:36 slave1.lc java[2060]: Caused by: com.spotify.docker.client.shaded.javax.ws.rs.client.ResponseProcessingException: com.spotify.docker.client.shaded.com.fasterxml.jackson.databind.JsonMappingException: Instantiation of [simple type, class com.spotify.docker.client.messages.MemoryStats$Stats] value failed: Null totalWriteback (through reference chain: com.spotify.docker.client.messages.ContainerStats["memory_stats"]->com.spotify.docker.client.messages.MemoryStats["stats"])
Feb 23 03:59:36 slave1.lc java[2060]: at org.glassfish.jersey.client.JerseyInvocation.translate(JerseyInvocation.java:806) ~[metrics.jar:DEV]
Feb 23 03:59:36 slave1.lc java[2060]: at org.glassfish.jersey.client.JerseyInvocation.access$700(JerseyInvocation.java:92) ~[metrics.jar:DEV]
Feb 23 03:59:36 slave1.lc java[2060]: at org.glassfish.jersey.client.JerseyInvocation$5.completed(JerseyInvocation.java:773) ~[metrics.jar:DEV]
Feb 23 03:59:36 slave1.lc java[2060]: at org.glassfish.jersey.client.ClientRuntime.processResponse(ClientRuntime.java:198) ~[metrics.jar:DEV]
Feb 23 03:59:36 slave1.lc java[2060]: at org.glassfish.jersey.client.ClientRuntime.access$300(ClientRuntime.java:79) ~[metrics.jar:DEV]
Feb 23 03:59:36 slave1.lc java[2060]: at org.glassfish.jersey.client.ClientRuntime$2.run(ClientRuntime.java:180) ~[metrics.jar:DEV]
Feb 23 03:59:36 slave1.lc java[2060]: at org.glassfish.jersey.internal.Errors$1.call(Errors.java:271) ~[metrics.jar:DEV]
Feb 23 03:59:36 slave1.lc java[2060]: at org.glassfish.jersey.internal.Errors$1.call(Errors.java:267) ~[metrics.jar:DEV]
Feb 23 03:59:36 slave1.lc java[2060]: at org.glassfish.jersey.internal.Errors.process(Errors.java:315) ~[metrics.jar:DEV]
Feb 23 03:59:36 slave1.lc java[2060]: at org.glassfish.jersey.internal.Errors.process(Errors.java:297) ~[metrics.jar:DEV]
Feb 23 03:59:36 slave1.lc java[2060]: at org.glassfish.jersey.internal.Errors.process(Errors.java:267) ~[metrics.jar:DEV]
Feb 23 03:59:36 slave1.lc java[2060]: at org.glassfish.jersey.process.internal.RequestScope.runInScope(RequestScope.java:340) ~[metrics.jar:DEV]
Feb 23 03:59:36 slave1.lc java[2060]: at org.glassfish.jersey.client.ClientRuntime$3.run(ClientRuntime.java:210) ~[metrics.jar:DEV]
Feb 23 03:59:36 slave1.lc java[2060]: at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) [na:1.8.0_77]
Feb 23 03:59:36 slave1.lc java[2060]: at java.util.concurrent.FutureTask.run(FutureTask.java:266) [na:1.8.0_77]
Feb 23 03:59:36 slave1.lc java[2060]: ... 3 common frames omitted

```

--- a/src/main/java/com/spotify/docker/client/messages/MemoryStats.java
+++ b/src/main/java/com/spotify/docker/client/messages/MemoryStats.java
@@ -146,6 +146,7 @@ public abstract class MemoryStats {
     @JsonProperty("total_unevictable")
     public abstract Long totalUnevictable();
 
+    @Nullable
     @JsonProperty("total_writeback")
     public abstract Long totalWriteback();
 

